kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/test/timeout/CoroutinesTimeoutTest.kt

```diff
@@ -88,3 +95,20 @@ private suspend fun someCoroutine() {
       }
    }
 }
+
+suspend fun nonVirtualTimeDelay(duration: Duration) {
```

Comment:
At your discretion: Would `realTimeDelay` be a better fit for our human wetware, avoiding negativity?